### ISSUE TITLE
Add optional brackets to Scheme syntax

### ISSUE THREAD

**Issue by jcubic** [PrismJS/prism#2811]
**Information**
- Language: Scheme
- Plugins: none

**Description**
Some books about Scheme show usage of brackets together with parenthesis. Example is [The Scheme Programming Language by Kent Dybvig](https://www.scheme.com/tspl4/) which is kind of classic. The R7RS Scheme spec don't specify brackets but some Scheme implementation use them optionally, not only Racket, that was once know as  PTL Scheme and I think brackets were already there.

I think it would be good idea to add brackets the same as racket have (the code can be reused or copy/pasted), so the highlighting will work with wider range of code examples.

**Code snippet**

[Test page](https://prismjs.com/test.html#language=scheme&text=(let%20(%5Bx%2010%5D%0A%20%20%20%20%20%20%5By%2020%5D)%0A%20%20(%2B%20x%20y)))

<details>
<summary>The code being highlighted incorrectly.</summary>

```
(let ([x 10]
      [y 20])
  (+ x y))
```
</details>

**Comment by RunDevelopment** [comment]
So brackets and parenthesis can be used interchangeably in some Scheme implementations?

**Comment by jcubic** [comment]
Yes, that's right. This is how for instance it work in BiwaScheme even that it's not required by the Scheme standard, the same happen with lisp macros (define-macro) they are not part of the standard but most scheme implementation have them.

It's good to have in generic scheme syntax highlighter.
